faqs/galaxy/datasets_moving_data_between_galaxy_servers.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Moving data between Galaxy servers
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+For individual datasets:
+1. Click on name of file (dataset) to expand the info.
+2. Right click on save icon {% icon galaxy-save %} > Copy Link. Note that for BAM files, clicking on save icon {% icon galaxy-save %} provides access to both alignment (BAM) and index (BAI) files. For BAM files use left mouse click to access BAM and index files. Move the cursor over the BAM file and use the right click to copy the link.
```

Comment:
This Note about BAM files could probably just be it's own bullet point. If possible, could you test how/if the link button works for BAM files and how to get the index file? Since I am not sure about that myself

Reply:
Okay I will do that. Thanks

Reply:
1. The "copy" link for BAM data is just the single BAM file. Other Galaxy server's don't need the index, they make their own index when a BAM is loaded (uploaded, copies, etc)

2. The "disc" icon has two datasets inside it -- for downloading. The BAM and the BAM.bai index. People need both when moving out of Galaxy to command-line or other applications. Not needed for Galaxy-to-Galaxy.

The first used to have to be done with the second (copy the link(s) out of the disc icon). Now it is broken out and pushed up to the link icon (and is still nested under the disk, just takes more clicks..).  I suggested dropping all of the "BAM" specific content here since it isn't needed. Can address how BAM (and other data with indexes) when discussing download or in datatypes. Example: fasta has an index too -- we just don't save it to the history at all (created and deleted at runtime)-- not even available as a nested data unless you specifically create it and save it to the history (never needed in Galaxy, but in case it comes up later --- it happens under pencil > edit attributes > convert > create fa.fai index). 

Moving a single datasets is just using that "link" copy in the "from" galaxy, then a paste into the upload tool at the "to" galaxy. The limitations I know of. Could mention both, or just the first. 

1) cannot move HTML datasets (security reasons) -- can only download
2) cannot move SQLite datasets (the db under the dataset is not what is actually copied) -- and these don't download either afaik